src/main/java/seedu/address/logic/commands/UnarchiveCommand.java

```diff
@@ -0,0 +1,71 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.ArchiveStatus;
+import seedu.address.model.person.Person;
+
+
+/**
+ * Unarchives a person identified using it's displayed index from the address book.
+ */
+public class UnarchiveCommand extends Command {
+
+    public static final String COMMAND_WORD = "c-unarchive";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Unarchive the person identified by the index number used in the displayed person list.\n"
```

Comment:
Should be `Unarchives` as the command is singular

Reply:
Thanks for pointing out, will change it.